src/sentry/utils/validators.py

```diff
@@ -11,6 +11,8 @@
 
 HEXADECIMAL_16_DIGITS = re.compile("^[0-9a-fA-F]{16}$")
 
+VALID_EMAIL_SEARCH = re.compile(r"(^[a-zA-Z0-9*_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)")
```

Comment:
nitpick:

I love reading regex'es just like the next person, but we tend to miss things so often helpful, besides the descriptive name as you added, a comment:

```suggestion
# Matches a valid email address, including a wildcard * as username
VALID_EMAIL_SEARCH = re.compile(r"(^[a-zA-Z0-9*_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)")
```